### PR TITLE
Bump identity-test-users version to avoid eviction warnings

### PR DESCRIPTION
**Pull request by davidfurey**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -4,7 +4,7 @@ object Dependencies {
 
   val play = "com.typesafe.play" %% "play" % "2.6.7"
   val identityCookie = "com.gu.identity" %% "identity-cookie" % "3.80"
-  val identityTestUsers = "com.gu" %% "identity-test-users" % "0.6"
+  val identityTestUsers = "com.gu" %% "identity-test-users" % "0.7"
```
Comment (rtyley) [inline project/Dependencies.scala:7]: I think the PR title should be `identity-test-users` not `identity-cookie`, based on this?

**Review by rtyley — APPROVED** [review]
Go for it!

File: project/Dependencies.scala (hunk 1 above)
Comment (davidfurey) [reply project/Dependencies.scala:7]: Oops.  Thanks.

**Comment by Mullefa** [comment]
Closed as this repo is getting archived once [#1571](https://github.com/guardian/identity/pull/1571) in identity is merged.

**Pull request closed without merging**